src/command/Menus.js

```diff
@@ -1067,7 +1067,11 @@ define(function (require, exports, module) {
         }
 
         // Add a file-menu class to hide non-file items from file context menu
-        $("#project-context-menu").toggleClass("file-menu", mouseOrLocation.fileMenu);
+        if(mouseOrLocation.fileMenu) {
```

Comment:
ok sounds good! Yeah I would say change it to the single line change (`!!`) and add a comment above it explaining that we need to force a boolean